README.md

```diff
@@ -142,6 +142,7 @@ For convenience aliases have been provided for all supported request methods.
 ##### axios.post(url[, data[, config]])
 ##### axios.put(url[, data[, config]])
 ##### axios.patch(url[, data[, config]])
+##### axios.options(url[, data[, config]])
```

Comment:
The `options` method does not accept `data` parameter. Could you please remove `data` and place `options` between `head` and `post` in `README.md`?